client/src/main/java/uk/co/blackpepper/bowman/MethodHandlerChain.java

```diff
@@ -4,9 +4,12 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreType;
+
 import javassist.util.proxy.MethodFilter;
 import javassist.util.proxy.MethodHandler;
 
+@JsonIgnoreType
```

Comment:
I had to add this in because when a resource that has a child resource is serialised, an error is thrown by Jackson - it tries to serialise this part of the proxy object and dies.
Ping me if you want to discuss or see a demo.